### DISCUSSION TITLE
Welcome to rhcsa8env Discussions!

### DISCUSSION THREAD

**Discussion by rdbreak** [rdbreak/rhcsa8env#29] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lambert-art** [comment]
Good afternoon,

I just installed the virtual machines for rhcsa8env. I wanted to have, if possible, a suggestion on how to proceed for the rhcsa exam simulation as I would like to obtain the exam200 certification.

Thanks Luke
